remarkable-review/The End of Paper.md

```diff
@@ -0,0 +1,101 @@
+
+## Why the reMarkable has stolen my heart
+I say "holy smokes" way too often.
+
+The reMarkable tablet arrived one November day in the middle of me recording a [livestream](https://www.youtube.com/watch?v=SDWjvhd98DI). I decided I would unbox and document my first experience on the stream. The video speaks for itself:
+
+<iframe width="420" height="315" src="https://youtu.be/W7hWbSxIvns">
+</iframe>
+
+If you made it through my fifteen utterances of "holy smokes" in seventy-five seconds, you also got to see the magic moment where I realized that this remarkable little tablet was going to replace my paper notebook.
+
+### It feels like writing on paper
+As I mentioned in the video, writing on reMarkable feels extremely close to writing on paper. It's difficult to describe without experiencing it. The combination of reMarkable's marker scraping across the tablet surface produces two distinct sensations:
+
+**Texture.** reMarkable eschews the typical "hard plastic on glass" feeling of writing on an iPad with the Apple Pencil. Instead, it feels like the fine grain of a lead pencil scraping across a lightly-textured paper. The resistance is delightful. I feel more accurate with each stroke, more connected to the surface.
+
+**"Sinkiness."** Somehow, I feel like I'm actually pressing _into_ the surface of the tablet with the tip of the marker. There's _give_ to it.
+
+**No (or very little) glare.** reMarkable's screen looks great in full sunlight. The surface of reMarkable is an e-paper screen, similar to the Kindle, but slightly more matte in appearance. Unlike the iPad or a computer screen, it reacts like paper. Side benefit: no blue light at night!
+
+### It does one thing well
+You can only do three things with reMarkable: write, read (and annotate) a PDF, or read (and annotate) an eBook. That's it.
+
+There's no email. No web browser. No social features. No notifications. You can sync your writings, PDFs, and eBooks to the cloud via WiFi, if you choose. There's not even a clock.
+
+Some reviewers complained about reMarkable's price tag given it can _only_ be used for writing and reading. They are entirely missing the point. In a world of sub-hundred dollar tablets that run every app under the sun, reMarkable's focused feature set sets it apart.
+
+Paradoxically, I'm perfectly willing to pay more for a device that does less. Rather than getting pounded into the ground by the incessant notifications of the digital domain, reMarkable provides a serene focus on the few things that actually matter, while still giving me the advantages of digital content.
+
+### It feels built-to-last
+I took my paper notebooks everywhere. They got lugged around in a stack of books, tossed onto tables, unceremoniously buried in backpacks. They got crumbs on them while I scrawled an idea while eating lunch, and spilled on by careless children. reMarkable's metal case, fabric folio, and light weight feel up to the challenges of everyday life.
```

Comment:
Maybe add something like "Unlike fragile paper, reMarkable's metal case fabric folio feel up to the challenges of everyday life."